Sources/_RegexParser/Regex/Parse/SyntaxOptions.swift

```diff
@@ -58,9 +58,14 @@ public struct SyntaxOptions: OptionSet {
   /// `(_: .*)` == `(?:.*)`
   public static var experimentalCaptures: Self { Self(1 << 5) }
 
+  /// The syntax kind of a multi-line literal. This will always be set when
+  /// parsing a multi-line literal. Note this does not imply extended syntax,
+  /// as that may be temporarily disabled while parsing.
+  public static var multilineLiteral: Self { Self(1 << 6) }
```

Comment:
This also cannot be un-set and there's no syntax for setting it in the regex. I wonder if it should be a let on the parser or something like that instead of an option. Not sure, thoughts?

Reply:
Talked offline, and decided to keep this as-is for now